code/Documentation/SchwurblAir/conf.py

```diff
@@ -0,0 +1,93 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+#import os
+#import sys
+#sys.path.insert(0, os.path.abspath('.'))
+
+from sphinx.builders.html import StandaloneHTMLBuilder
+import subprocess, os
+
+# Doxygen
+subprocess.call('doxygen Doxyfile.SchwurblAir', shell=True)
+
+# -- Project information -----------------------------------------------------
+
+project = 'SchwurblAir'
+copyright = '2020, Mommel'
```

Comment:
Codacy found an issue: [Redefining built-in 'copyright'](https://app.codacy.com/gh/mommel/hs-lr-midi-schwurbler/pullRequest?prid=6419091)